projects/plugins/jetpack/modules/publicize/publicize-jetpack.php

```diff
@@ -448,25 +616,37 @@ public function post_is_done_sharing( $post_id = null ) {
 			return false;
 		}
 
-		return 'publish' == $post->post_status || get_post_meta( $post->ID, $this->POST_DONE . 'all', true );
+		return 'publish' === $post->post_status || get_post_meta( $post->ID, $this->POST_DONE . 'all', true ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
 	}
 
 	/**
 	 * Save a flag locally to indicate that this post has already been Publicized via the selected
 	 * connections.
+	 *
+	 * @param int      $post_ID Post ID.
+	 * @param \WP_Post $post Post object.
 	 */
-	function save_publicized( $post_ID, $post = null, $update = null ) {
+	public function save_publicized( $post_ID, $post = null ) {
```

Comment:
I am assuming we don't need the thid param and this is why we did this earlier in the PR 

```
add_action( 'wp_insert_post', array( $this, 'save_publicized' ), 11, 2 );

```

Is it possible `wp_insert_post` could have been hooked into other places? For example, can I use an existing version of the plugi and using code snippeters hook on to `wp_insert_post`? If yes, should just let this be the way it orginally was?

Reply:
The only problem could be if someone is calling `save_publicized` directly with 3 parameters, but in that case I think they would just get a warning. They also shouldn't be doing that, especially as the parameter isn't used 😄